tests/integration/list/list_integ_base.py

```diff
@@ -0,0 +1,29 @@
+import os
+from unittest import TestCase
+from pathlib import Path
+import uuid
+import shutil
+import tempfile
+from tests.testing_utils import get_sam_command
+
+
+class ListIntegBase(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.cmd = cls.base_command()
+        cls.test_data_path = Path(__file__).resolve().parents[1].joinpath("testdata", "list")
+
+    def setUp(self):
+        super().setUp()
+        self.scratch_dir = str(Path(__file__).resolve().parent.joinpath(str(uuid.uuid4()).replace("-", "")[:10]))
+        os.mkdir(self.scratch_dir)
+        self.working_dir = tempfile.mkdtemp(dir=self.scratch_dir)
+
+    def tearDown(self):
+        super().tearDown()
+        self.working_dir and shutil.rmtree(self.working_dir, ignore_errors=True)
+        self.scratch_dir and shutil.rmtree(self.scratch_dir, ignore_errors=True)
```

Comment:
Regarding to our discussion, we don't need any of these lines starting from line 14 to line 25, we can just remove them :)

Reply:
Sorry, misinterpreted your earlier comment, I've removed them